subsys/spm/Kconfig

```diff
@@ -168,237 +169,28 @@ config SPM_BLOCK_NON_SECURE_RESET
 	  service. If not enabled, a debugger will not be able to issue a
 	  system reset while the core is executing Non-Secure code.
 
-menu "Configure Non-Secure peripherals"
-
-config SPM_NRF_CLOCK_NS
-	bool "Clock control is Non-Secure"
-	default y
-
-config SPM_NRF_RTC0_NS
-	bool "RTC0 is Non-Secure"
-	default y
-
-config SPM_NRF_RTC1_NS
-	bool "RTC1 is Non-Secure"
-	default y
 
 config NRF_RTC_TIMER
-	depends on !SPM_NRF_RTC1_NS
-	help
-	  RTC1 cannot be used the system clock driver for SPM if it is Non-Secure
+	# RTC1 cannot be used the system clock driver for SPM if it is not Secure
```

Comment:
Why is this moved from help text to a comment?

Reply:
Will revert.